src/main/js/CreatePage.js

```diff
@@ -0,0 +1,306 @@
+import React, { useState, useEffect } from 'react';
+import Card from '@material-ui/core/Card';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import DeleteIcon from '@material-ui/icons/Delete';
+import Tooltip from '@material-ui/core/Tooltip';
+import HelpIcon from '@material-ui/icons/Help';
+import './SurveyPage.scss';
```

Comment:
We should rename the style sheet. Did any of the styles break? Since we split up the components but not the stylesheet.